### ISSUE TITLE
Perfect synchronization with multiple audios 

### ISSUE THREAD

**Issue by JoanIzquierdo** [jplayer/jPlayer#392]
Is it possible to synchronize several sounds mp3/ogg? when you modify the progress bar, the two sounds must be syncronized.

**Comment by martindefatte** [comment]
Yes, at one point I had this working on this page... but for some reason it's broken in Chrome now. I believe it still works in Safari. Feel free to investigate.
http://www.mysteryroommastering.com/samples/
